binancial/data/get_all_pairs.py

```
@@ -2,9 +1,9 @@
+def get_all_pairs(client):
```

Ruff (0.8.2)

Missing return type annotation for public function `get_all_pairs`

(ANN201)